### ISSUE TITLE
Remove print statements before releasing on PyPi

### ISSUE THREAD

**Issue by jochasinga** [mc2-project/mc2#179]
There is current a few places where there are still `print` statements. Can we switch to `logger` instead?

https://github.com/mc2-project/mc2/blob/bfbec919c0e10a48b3655a4cde5b5a5640f9ef58/python-package/tests/azure/delete_azure_resources.py#L6

**Comment by jochasinga** [comment]
@chester-leung are these print statements save to remove and use logger instead?

**Comment by chester-leung** [comment]
Yes, we can replace these print statements with log messages. Currently I don't believe there's a logger configured for the tests as a whole though.
